lib/Utils/Backends.pm

```diff
@@ -83,4 +84,9 @@ sub is_svirt_except_s390x {
 
 }
 
+sub is_spvm_backend {
+    # PowerVM backend, poo#33388
```

Comment:
Please properly add the documentation, (check #7495).

Reply:
done, thanks